geode-docs/managing/security/method_invocation_authorizers.html.md.erb

```diff
@@ -0,0 +1,165 @@
+this authorizer implementation addresses all four security risks: `Java Reflection`, `Cache Modification`, `Region Modification` and `Region Entry Modification`.
+It should be noted, however, that the `Region Entry Modification` security threat might be re-introduced: users with the `DATA:READ:RegionName` privilege will be able to execute any method whose name 
+starts with `is` or `get` on the objects stored within the region and on instances used as bind parameters, providing they are in the pre-configured packages. If those methods do not fully follow 
+the [JavaBean Specification 1.01](https://download.oracle.com/otndocs/jcp/7224-javabeans-1.01-fr-spec-oth-JSpec/) in that **accessors do not mutate the object state**, then instances could be potentially modified in place.
+
+**Note:**
+Usage of this authorizer is only recommended for secured clusters on which the user has full confidence in that all objects stored within the regions and used as bind parameters follow the 
+[JavaBean Specification 1.01](https://download.oracle.com/otndocs/jcp/7224-javabeans-1.01-fr-spec-oth-JSpec/).
+It might also be used on clusters on which all entries stored are immutable.
+
+### <a id="regExMethodAuthorizer"></a>RegExMethodAuthorizer
+
+A fully flexible `MethodInvocationAuthorizer` that allows methods to be invoked during the query execution only if the the following conditions are met:
+
+- The method is not considered permanently forbidden by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+- The method does not belong to <%=vars.product_name%>, or does belong but it's considered safe by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+- The fully qualified method name matches at least one of the pre-configured regular expressions.
+
+When correctly configured, this authorizer implementation addresses the four main security risks: `Java Reflection`, `Cache Modification`, `Region Modification` and `Region Entry Modification`.   
+For the statement to remain true, however, the regular expressions used must be exhaustively studied and configured so no mutator methods ever match. If the regular expressions are not restrictive enough, 
+the `Region Entry Modification` security risk might be potentially re-introduced: users with the `DATA:READ:RegionName` privilege will be able to execute methods (even those modifying the entry) on 
+the objects stored within the region and on instances used as bind parameters of the query. 
+
+**Note:**
+This authorizer must be used with extreme care, it is the most powerful in terms of flexibility and versatility (full control through regular expressions regarding what to allow and what to forbid); 
+but it's also the most dangerous as one small mistake in the configured regular expressions can wronly allow a wide variety of non safe methods to be executed.
```

Comment:
Typo in "wrongly." Also, it might be better to use "unexpectedly" or something similar.

Reply:
👍